### PR TITLE
Update to wgpu 0.10

### PR DESCRIPTION
**Pull request by rukai**
wgpu is looking to release 0.10 in a few weeks, once that happens I'll update this PR to use the crates.io release instead of git master.

### CODE REVIEW

**Comment by rukai** [comment]
wgpu 0.10 is released we should land this now

File: examples/hello.rs [hunk 1]
```diff
@@ -94,10 +94,11 @@ fn main() -> Result<(), Box<dyn Error>> {
...
                 // Get the next frame
-                let frame = swap_chain
-                    .get_current_frame()
-                    .expect("Get next frame")
-                    .output;
+                let frame =
+                    surface.get_current_frame().expect("Get next frame").output;
+                let view = &frame
+                    .texture
+                    .create_view(&wgpu::TextureViewDescriptor::default());
```
Comment (hecrj) [inline examples/hello.rs:101]: Is this the proper way to render now? Do we have to explicitly create a view for each frame texture or is there a better way?

**Review by hecrj — COMMENTED** [review]
Looks great overall! I just have a couple of questions.

Thank you again, @rukai 🙇

File: Cargo.toml [hunk 2]
```diff
@@ -9,9 +9,10 @@ keywords = ["font", "ttf", "truetype", "wgpu", "text"]
 repository = "https://github.com/hecrj/wgpu_glyph"
 documentation = "https://docs.rs/wgpu_glyph"
 readme = "README.md"
+resolver = "2"
```
[the remark this answers is not in the document]
Comment (rukai) [reply Cargo.toml:12]: removing it and trying to compile will demonstrate the issue.
```
error: Metal API enabled on non-Apple OS. If your project is not using resolver="2" in Cargo.toml, it should.
  --> /home/rukai/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/4dc5546/wgpu-hal/src/lib.rs:45:1
   |
45 | compile_error!("Metal API enabled on non-Apple OS. If your project is not using resolver=\"2\" in Cargo.toml, it should.");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: DX12 API enabled on non-Windows OS. If your project is not using resolver="2" in Cargo.toml, it should.
  --> /home/rukai/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/4dc5546/wgpu-hal/src/lib.rs:47:1
   |
47 | compile_error!("DX12 API enabled on non-Windows OS. If your project is not using resolver=\"2\" in Cargo.toml, it should.");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Seems like the wgpu project has decided to depend on resolver = 2 being enabled.

File: examples/hello.rs (hunk 1 above)
Comment (rukai) [reply examples/hello.rs:101]: Yeah seems like it.
Its what all the examples are doing:
* https://github.com/gfx-rs/wgpu/blob/564330432f60b967a69b36a0094d72e1e77ebb72/wgpu/examples/framework.rs#L302
* https://github.com/gfx-rs/wgpu/blob/564330432f60b967a69b36a0094d72e1e77ebb72/wgpu/examples/hello-triangle/main.rs#L101

File: Cargo.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (hecrj) [reply Cargo.toml:12]: Ah, interesting... So it has to be enabled all the way down the dependency tree...

File: examples/hello.rs (hunk 1 above)
Comment (hecrj) [reply examples/hello.rs:101]: Great!

**Review by hecrj — APPROVED** [review]
Thanks for the quick turnaround! Let's merge and release! 🚢

**Pull request merged**